app/controllers/jokes.js

```diff
@@ -0,0 +1,6 @@
+const jokes = require('../services/jokes');
+
+exports.jokes = async (req, res) => {
+  const joke = await jokes.getJokes();
+  res.status(200).send(joke);
```

Comment:
What happens if jokes service returns an error? The controller always returns status 200?

Reply:
That error shouldn't be catch from getJokes? so doesn't will be redundant to return an error there too?

Reply:
But if you see, you are returning the error object so the joke variable will have the error and then you returning the error with 200